db/schema.rb

```diff
@@ -42,6 +42,7 @@
     t.jsonb "json_data"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.jsonb "sva_json_data"
```

Comment:
wozu brauchen wir ein abweichendes/zusätzliches Feld `sva_json_data` wenn wir `json_data` schon haben?

Reply:
sva_json_data ist das geparste jsondata. Da machen wir bei den anderen importer ja auch.

Reply:
und warum nutzt du nicht "json_data"?

Reply:
bitte `json_data` statt `sva_json_data` nutzen

Reply:
in json_data, wir das original von der maz gespeichert und in sva_json_data das geparste. Ich finde es gut und wichtig das so zu machen, so kann man mögliche Fehler im Nachhinein leichter nachvollziehen.

Reply:
dann kann so bleiben